pyproject.toml

```diff
@@ -210,6 +210,9 @@ exclude = [
 ]
 namespaces = true
 
+[tool.setuptools.exclude-package-data]
+"*" = ["ruff.toml"]
```

Comment:
Let's add a comment saying why this is here. Because it doesn't mention `_vendor`, there's nothing linking to its purpose.
```suggestion
# Remove ruff.toml when installing vendored packages (#4652)
"*" = ["ruff.toml"]
```